src/components/editor/EditorReadMe.jsx

```diff
@@ -0,0 +1,36 @@
+<p><br />Step 3: Admin<br />BE CAREFUL, THIS IS WHERE THINGS CAN GO WRONG<br />* Go to admin.thegazelle.org (same as where you uploaded images)<br />* Click on Articles<br />* Add author(s), categories and make sure that there is a image URL and a teaser (these should come through from Ghost if you&rsquo;ve done the above correctly). If there is not, copy from Ghost.<br />* Click Save Changes<br /> Screen Shot 2017-05-03 at 2.49.55 pm.png</p>
+<p>* To create a new author<br /> * click on Authors<br /> * Search the person&rsquo;s name to double-check there hasn&rsquo;t been another entry for that person in the past<br /> * Add name how you would like it to appear on the website<br /> * Add slug: first-last &lt; notice no caps, and hyphen between names<br />* Click on Issues<br />* Create a new issue (automatically generates unless it&rsquo;s a special issue)<br />* Click on Articles<br />* Click on Search By List<br />* Add ONE Featured Article and TWO Editor&rsquo;s Picks<br />* Add the rest of the articles<br />* Click Save Changes<br /> Screen Shot 2017-05-03 at 2.54.27 pm.png</p>
+<p>&nbsp;</p>
+<p>* Click Save Changes<br />* Click on Categories &gt; order the categories as you like<br />* Click on Main &gt; Publish Issue &gt; Restart Servers<br /> * You&rsquo;ll get an unknown error pop-up, click ok and you should have a brand new Gazelle issue at thegazelle.org</p>
+<p><br />Trouble Shooting<br />* Click on RESTART SERVERS<br />* Refresh the page (and login again)<br />* Clear your cache<br />* Make sure every article you want to publish has a category, author, image and teaser assigned to it.<br /> * Go back to Ghost and check you&rsquo;ve uploaded both images<br />* Make sure every image starts with https://<br />* Cry<br />* If crying didn&rsquo;t work, contact Emil or whoever is currently familiar with the back-end of The Gazelle</p>
+<p><br />BIG NO NOS:<br />* Deleting a post in Ghost that was already published as an article and is assigned to an issue &mdash; this would crash the website by corrupting the database<br />* Publishing an issue with no articles in it or less than 4<br />* Unpublishing an article, in admin.thegazelle.org without removing it from the issue(s) it is in (this could potentially also crash the website by corrupting the database)</p>
+<p><br />Good form:<br />* Unpublish articles that you removed from all issues (if they are published the URLs to them will still work, though it wouldn&rsquo;t crash the website, just direct URLs to them would still show up instead of our 404 Page Not Found)</p>
+      </div>
```

Comment:
And a `/* eslint-enable */` comment here.

Normally we only disable the linter when it's really needed, but this is one of the cases when it's fine. And of course it's also temporary.